app/controllers/annotations_controller.rb

```diff
@@ -34,7 +34,7 @@ def add_existing_annotation
         page: params[:page],
         **base_attributes
       )
-    elsif submission_file.is_pynb?
+    elsif submission_file.is_pynb? || submission_file.is_rmd?
```

Comment:
The other major thing I realized  about this feature is that the presence of annotations means this change is not strictly backwards compatible, since current Rmd files use `TextAnnotation`s but this change makes them use `HtmlAnnotation`s. This is not good and I should have realized this sooner!

To move forward with this PR for now please do the following: add a "feature flag" configuration option `rmd_convert_enabled`, and add guards everywhere so that the changed code paths check whether this option is `true`, similar to what we're doing for the `nbconvert_enabled` option. If the option is `false`, the current behaviour (showing the Rmd files raw) should be used instead. Look at `config/settings.yml` and `config/application.rb` for examples on adding configuration options.